asketch2sketch/helpers/fixSVG.js

```diff
@@ -13,6 +13,7 @@ function makeNativeSVGLayer(layer) {
   svgLayer.frame().setY(layer.y);
   svgLayer.frame().setWidth(layer.width);
   svgLayer.frame().setHeight(layer.height);
+  svgLayer.resizingConstraint = layer.resizingConstraint;
```

Comment:
shouldn't we set this only if `resizesContent === 1` 🤔

Reply:
It doesn't seem to matter and I'm removing the `resizesContent` property from base.